kafka-client/src/main/java/dev/responsive/kafka/internal/stores/PartitionedOperations.java

```diff
@@ -182,14 +192,16 @@ public static PartitionedOperations create(
   }
 
   private static RemoteKVTable<?> createInMemory(final ResponsiveKeyValueParams params) {
+    // TODO(sophie): implement ttl for in-memory store?
```

Comment:
same as above (remove TODO and create a ticket) we should probably throw an exception though if there is a non-empty TTL Resolver

Reply:
https://www.notion.so/responsivedev/Support-ttl-for-in-memory-store-12927549be4c80199207f894273c0b03?pvs=4